diagnostics/EOF_500hPa/compute_anomalies.ncl

```diff
@@ -21,13 +22,10 @@ time_coord = getenv("time_coord")
 lat_coord = getenv("lat_coord")
 lon_coord = getenv("lon_coord")
 lev_coord = getenv("lev_coord")
-hyam_var = getenv("hyam_var")
-hybm_var = getenv("hybm_var")
-
 file_path = getenv("DATADIR")
 
-zg_var = getenv("zg_var")
-ps_var = getenv("ps_var")
+print("IS LOG SHIT WORKING")
```

Comment:
Remove debugging statement

Reply:
Thanks; fixed in f9c261e32936b3cd8dd1c6181826a56a2b495816